modalstyle.css

```diff
@@ -0,0 +1,195 @@
+@media (max-width: 768px) {
```

Comment:
- [ ] Kindly fix the `Style` of the modal of the `Mobile` version. Your project style looks the same for the `Mobile` and `Desktop` versions. But in [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9), styles are different in both cases.
***Your Design***
![Screenshot from 2023-01-06 05-10-02](https://user-images.githubusercontent.com/60083437/210898012-434a1418-cbac-42c7-9337-295c858adaf4.png)
***Figma Design***
![Screenshot from 2023-01-06 05-14-31](https://user-images.githubusercontent.com/60083437/210898093-cb5a6c38-d198-4b65-ba19-2f3412cc1812.png)